python/python/glide/async_commands/cluster_commands.py

```diff
@@ -624,3 +624,25 @@ async def lolwut(
             TClusterResponse[str],
             await self._execute_command(RequestType.Lolwut, args, route),
         )
+
+    async def random_key(self, route: Optional[Route] = None) -> Optional[str]:
+        """
+        Returns a random key from currently selected database.
```

Comment:
Maybe
```suggestion
        Returns a random key.
```
because cluster installation has no databases.
Consider updating line 643 too